packages/webapi/src/inheritance.ts

```diff
@@ -1,4 +1,4 @@
-export const inheritence = {
+export const inheritance = {
```

Comment:
Let’s avoid actual code changes. Doesn’t really matter if an internal API is misspelled and changing it with an automated tool is risky without any particular benefit.

Reply:
I’ve split this out into #5634. It’s easy to review with `git grep inheritence` and `git ls-files | grep inheritence`.